File: ejercicios/c++/ejercicio069.cpp
#include <iostream>
using namespace std;

int factorial_recursivo(int numero){
	if(numero < 1)
		return 1;
	else
		return numero * factorial_recursivo(numero - 1);
}


int main() {
	int numero;
	cout << "ingrese un numero entero : ";
	cin >> numero;
	
	if (numero > 0)
		cout << numero << "! " << " = " << factorial_recursivo(numero) << endl;
	else
		cout << "El numero debe ser positivo." << endl;
	return 0;
}

File: ejercicios/python/ejercicio069.py
def factorial_recursivo(numero):
   if (numero < 1):
      return 1
   else:
      return numero * factorial_recursivo(numero - 1)

if __name__ == "__main__":
   numero = int(input("ingrese el numero : "))
   if (numero > 0):
      print (str(numero) + "!" + " = " + str(factorial_recursivo(numero)))
   else:
      print("El numero debe ser positivo.")
      
